extensions/wikia/AdEngine/AdEngine2.setup.php

```diff
@@ -21,6 +21,9 @@
 $wgAutoloadClasses['ResourceLoaderScript'] = __DIR__ . '/ResourceLoaders/ResourceLoaderScript.php';
 $wgAutoloadClasses['AdEngine2ApiController'] = __DIR__ . '/AdEngine2ApiController.class.php';
 
+// Recovery
+$wgAutoloadClasses['ResourceLoaderAdEngineBlockthroughCode'] = __DIR__ . '/ResourceLoaders/ResourceLoaderAdEngineBlockthroughCode.php';
```

Comment:
This repo is open ;p should we hide this name too?

Reply:
It's actually not visible on frontend, but why not. Renamed.